app/controllers/course/conditions_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Course::ConditionsController < Course::ComponentController
+  before_action :load_and_authorize_conditional
+  helper_method :return_to_path
+
+  # @return [String|Symbol] The path to return to when successfully executing or failing an action,
+  #         or when the user clicks on the `Back` button.
+  def return_to_path
```

Comment:
I agree that I almost always use the browser's back function. But we also need a return path on a successful action, don't we?